.github/workflows/publish_layer.yml

```diff
@@ -0,0 +1,207 @@
+name: Deploy layer to all regions
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  # TODO: remove after setup is finished
+  workflow_dispatch: { }
+  workflow_run:
+    workflows: [ "Publish to PyPi" ]
+    types:
+      - completed
+
+
+jobs:
+  build-layer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.12'
```

Comment:
Use `env` to make it easier to change it afterwards - example: https://github.com/awslabs/aws-lambda-powertools-python/blob/develop/.github/workflows/on_merged_pr.yml#L10:L10

Reply:
Not sure there is more value to extract it to a variable. We don't do it for python 3.8 version across workflows yet. It won't change frequently though and imho helps to read it more fluently and not scroll back to the top of the variable definition. What do you think?

Reply:
It's a nitpick (forgot to tag it accordingly). I agree. we don't do it elsewhere yet, so let's stay consistent. Resolving.